src/types/jsonrpc.ts

```diff
@@ -29,7 +29,18 @@ export const SignTransactionStruct = tuple([
   })
 ]);
 
-export type TSignTransaction = Infer<typeof SignTransactionStruct>;
```

Comment:
Why?

Reply:
The type for `to` is inferred as `string`, but it can also be undefined, causing type errors elsewhere.